src/devices/Card/CreditCard/CreditCard.cs

```diff
@@ -513,18 +552,17 @@ private bool FillTags()
                         var ret = ReadRecord(appSfi.Data[0], record);
                         LogInfo.Log($"Read record {record}, SFI {appSfi.Data[0]}, status: {ret}", LogLevel.Debug);
                     }
+
                     _alreadyReadSfi = true;
                 }
+
                 return false;
             }
         }
 
         /// <summary>
-        /// 
+        /// Read log records
         /// </summary>
```

Comment:
I'm surprised this builds correctly - I recall getting errors when params are not specified

Reply:
Right, that's possible. Maybe this depends on some project-wide warning settings as well? I sometimes get warnings about XML documentation checks being disabled (but didn't investigate on that yet)